component/background/RoomService.tsx

```diff
@@ -11,11 +12,16 @@ const config = {
 };
 firebase.initializeApp(config);
 
-class RoomService {
+const allRooms = firebase.database().ref("rooms");
+const allUsers = firebase.database().ref("users");
+
+const MESSAGE = "message";
+const USER = "user";
+
+class RoomService implements IRoomService {
   private rootRef: firebase.database.Reference;
   private messageRef: firebase.database.Reference;
-  private userListRef: firebase.database.Reference;
-  private myConfRef: firebase.database.Reference;
+  private myself: firebase.database.Reference;
```

Comment:
I am not quite sure what this variable is supposed to be for. I suggest a better name that is a little more descriptive.